collect.py

```diff
@@ -7,9 +7,11 @@
 from PIL import Image
 from selenium import webdriver
 from selenium.common.exceptions import NoAlertPresentException, NoSuchWindowException, TimeoutException
-
+from concurrent.futures import ThreadPoolExecutor
```

Comment:
The order of the imports should be:
1) Built-in imports
2) Imports from third-party libraries
3) Imports from our library

So this should go in the first part, before `json` to have them in alphabetical order.